package.json

```diff
@@ -70,18 +70,18 @@
     "karma-benchmark": "^0.4.0",
     "karma-chrome-launcher": "^0.2.0",
     "karma-firefox-launcher": "^0.1.6",
-    "karma-junit-reporter": "^0.2.2",
+    "karma-junit-reporter": "^0.3.3",
     "karma-safari-launcher": "^0.1.1",
-    "lodash": "~2.4.1",
-    "minifyify": "7.0.0",
+    "lodash": "~3.10.1",
```

Comment:
The difference is likely due to the shift in --save, --save-dev behavior based on the version of npm various developers were using, as opposed to anything conscious.

I'm thinking ^ (flexible minor version) is fine for devDeps (and we'd have to react ASAP to breakage if deps sneak in breaking changes with minor versions), but for product deps, we want to stay with fixed versions.